website/content/docs/configuration/replication.mdx

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+page_title: Replication - Configuration
+sidebar_title: <code>replication</code>
+description: |-
+  The replication stanza specifies various parameters for tuning replication related values.
+---
+
+# `replication` Stanza
+
+The `replication` stanza specifies various parameters for tuning replication related values.
+
+```hcl
+replication {
+  resolver_discover_servers = true
+  logshipper_buffer_length = 1000
+  logshipper_buffer_size = "5gb"
+}
+```
+
+## `replication` Parameters
+
+- `resolver_discover_servers` `(boolean: true)` - Indicates if the resolver should discover servers.
```

Comment:
It's probably fine to leave this as-is, but I could see it leading to people opening issues to ask for more details.  To pre-empt that, we could explain:

This feature should probably always be turned on.  It enables secondary cluster nodes to reach out to nodes in the primary cluster to request information on who the active node is. This fixes an issue where a secondary cluster may only know about a single node in the primary cluster, and if that node isn't the active node the replication connection will not succeed. With resolving disabled, the only way around this is to go through an onerous UpdatePrimary workflow, or do a step-downs on the primary cluster until the one node known by the secondary cluster is active.

Reply:
Thank you for this. I wasn't entirely sure what to put here. 😅